tests/testthat/test-async-cluster.R

```diff
@@ -0,0 +1,28 @@
+# Run this test for users to make sure the async cluster creation features
+# of doAzureParallel are still working
+context("async cluster scenario test")
+test_that("Async cluster scenario test", {
+  testthat::skip("Live test")
+  testthat::skip_on_travis()
+  credentialsFileName <- "credentials.json"
+  clusterFileName <- "test_cluster.json"
+
+  doAzureParallel::generateCredentialsConfig(credentialsFileName)
+  doAzureParallel::generateClusterConfig(clusterFileName)
+
+  # set your credentials
+  doAzureParallel::setCredentials(credentialsFileName)
+
+  clusterName <-
+    doAzureParallel::makeCluster(clusterSetting = clusterFileName, wait = FALSE)
+
+  while (is.null(getCluster(clusterName))) {
+    Sys.sleep(30)
```

Comment:
Is this long enough? Are you only checking that the cluster object was created in Batch, or that it is fully provisioned?

Reply:
updated getCluster to not check if the cluster is fully provisioned.